Realm/Realm/Handles/SharedRealmHandle.cs

```diff
@@ -238,6 +255,7 @@ public static void Initialize()
             DictionaryHandle.KeyNotificationCallback notifyDictionary = DictionaryHandle.NotifyDictionaryChanged;
             NativeMethods.MigrationCallback onMigration = OnMigration;
             NativeMethods.ShouldCompactCallback shouldCompact = ShouldCompactOnLaunchCallback;
+            NativeMethods.HandleTaskCompletionCallback handleWriteCommitAsync = HandleWriteCommitAsyncCallback;
```

Comment:
What is that you'd like to see? Something like `handleTaskCompletion`?

Reply:
Yeah - the var name should be closer to the type name.

Reply:
Then I prefer to change the type name to be more explicit about what it's supposed to handle. Is that ok with you?

Reply:
Isn't the same callback used for both begin and commit write?

Reply:
Yes, it is. That's why I'd rename the type to be `HandleWriteCommitAsync`

Reply:
I think `HandleTaskCompletion` is better in this case because `HandleWriteCommit` sounds like it will only handle the commit part, not the begin write part.